connector/fyle_integrations_platform_connector/apis/projects.py

```diff
@@ -23,7 +23,8 @@ def sync(self):
                     'attribute_type': self.attribute_type,
                     'display_name': self.attribute_type.replace('_', ' ').title(),
                     'value': project['name'],
-                    'source_id': project['id']
+                    'source_id': project['id'],
+                    'active': project['is_enabled']
```

Comment:
This will always be true - since you are only pulling active projects
Check query params
```json
query_params={"is_enabled": "eq.true"})
```